cypress/tests/integration/ArticleViewDCMetadata.spec.js

```diff
@@ -0,0 +1,159 @@
+		<meta name="DC.Date.issued" scheme="ISO8601" content="2022-10-19"/>
+		<meta name="DC.Date.modified" scheme="ISO8601" content="2022-10-19"/>
+		<meta name="DC.Description" xml:lang="en" content="The signaling theory suggests that dividends signal future prospects of a firm. However, recent empirical evidence from the US and the Uk does not offer a conclusive evidence on this issue. There are conflicting policy implications among financial economists so much that there is no practical dividend policy guidance to management, existing and potential investors in shareholding. Since corporate investment, financing and distribution decisions are a continuous function of management, the dividend decisions seem to rely on intuitive evaluation."/>
+		<meta name="DC.Format" scheme="IMT" content="application/pdf"/>
+		<meta name="DC.Identifier" content="mwandenga-signalling-theory"/>
+		<meta name="DC.Identifier.pageNumber" content="71-98"/>
+		<meta name="DC.Identifier.URI" content="http://localhost:8000/index.php/publicknowledge/article/view/mwandenga-signalling-theory"/>
+		<meta name="DC.Language" scheme="ISO639-1" content="en"/>
+		<meta name="DC.Rights" content="Copyright (c) 2022 Journal of Public Knowledge"/>
+		<meta name="DC.Rights" content=""/>
+		<meta name="DC.Source" content="Journal of Public Knowledge"/>
+		<meta name="DC.Source.ISSN" content="0378-5955"/>
+		<meta name="DC.Source.Issue" content="2"/>
+		<meta name="DC.Source.Volume" content="1"/>
+		<meta name="DC.Source.URI" content="http://localhost:8000/index.php/publicknowledge"/>
+		<meta name="DC.Subject" xml:lang="en" content="Professional Development"/>
+		<meta name="DC.Subject" xml:lang="en" content="Social Transformation"/>
+		<meta name="DC.Title" content="The Signalling Theory Dividends: A Review Of The Literature And Empirical Evidence"/>
+		<meta name="DC.Type" content="Text.Serial.Journal"/>
+		<meta name="DC.Type.articleType" content="Articles"/>
+	*/
+	it('Tests if Header DC Metadata are present and consistent', function() {
+		var issueTitle = 'Vol. 1 No. 2 (2014)';
+
+		cy.login('admin', 'admin');
+		cy.get('a').contains('admin').click();
+		cy.get('a').contains('Dashboard').click();
+		cy.get('.app__nav a').contains('Issues').click();
+
+		cy.visit('index.php/publicknowledge/manageIssues#back');
+		cy.get('span:contains("' + issueTitle + '")').prev('a.show_extras').click();
+		cy.get('tr:contains("' + issueTitle + '")').next().contains('a', 'Unpublish Issue').click();
+		cy.get('button:contains("OK")').click();
+
+		cy.logout();
+		cy.login('admin', 'admin');
+
+		cy.visit('index.php/publicknowledge/manageIssues#future');
+		cy.get('span:contains("' + issueTitle + '")').prev('a.show_extras').click();
+		cy.get('tr:contains("' + issueTitle + '")').next().contains('a', 'Publish Issue').click();
+		cy.get('input[id="sendIssueNotification"]').click();
+		cy.get('button[id^=submitFormButton]').click();
+
+		cy.logout();
```

Comment:
This does not have consequences for other articles, right?
Would be maybe be better to unpublish/re-publish just that submission? 
:thinking: